back-end/java/src/DormRoom/CampusLocation.java

```diff
@@ -0,0 +1,24 @@
+package DormRoom;
+
+/**
+ * An enum representing general campus areas where a dorm room
+ * building can be located.
+ */
+public enum CampusLocation {
```

Comment:
We will need to update this later with all campus locations. Maybe soon even though the sample dataset doesn't include all locations? I can send a list of all locations if needed.

Reply:
Yes, please add a commit adding other buildings to the DormBuilding switch case.